docs/getting-started/quickstart.md

```diff
@@ -259,4 +256,18 @@ To see that it's working, visit this URL in your browser: [http://localhost:3000
 
 Congratulations!  You just launched your DBOS application locallly!
 
+### 4. View your App Records in the Database
```

Comment:
I wouldn't add this. It's a bunch of extra detail that isn't really specific to DBOS at all, and also installing `psql` can be a pain.

Reply:
I change this to a `tip` that only shows up in the docker section - to say
```
 You can connect to this container just like a local Postgres database, and run queries with common tools like [psql](https://www.postgresql.org/docs/current/app-psql.html). It accepts connections on `localhost`, the default port 5432, username `postgres` and the password you set above.
```
I think that's really the gap to bridge. No need to say anything to someone who already has their own postgres. Just a hint for someone using that docker for the first time